linkml_runtime/utils/schemaview.py

```diff
@@ -654,19 +681,20 @@ def in_schema(self, element_name: ElementName) -> SchemaDefinitionName:
         """
         ix = self.element_by_schema_map()
         if element_name not in ix:
-            raise ValueError(f"Element {element_name} not in any schema")
+            msg = f"Element {element_name} not in any schema"
+            raise ValueError(msg)
         return ix[element_name]
 
     @lru_cache(None)
     def element_by_schema_map(self) -> dict[ElementName, SchemaDefinitionName]:
         ix = {}
         schemas = self.all_schema(True)
         for schema in schemas:
-            for type_key in [CLASSES, SLOTS, TYPES, ENUMS, SUBSETS]:
-                for k, v in getattr(schema, type_key, {}).items():
+            for type_key in ELEMENTS:
+                for k in getattr(schema, type_key, {}):
```

Comment:
value is not used, so no need for `.items()`

Reply:
I picked the changes for each commit manually using VSCode, and this was one that accidentally got split between commits.